avro_schema/utils.lua

```diff
@@ -53,8 +53,137 @@ function has_only(t, key)
     return fst_key == key and snd_key == nil
 end
 
+local function map(func, tab)
+    local res = {}
+    for k, v in pairs(tab) do
+        res[k] = func(k, v)
+    end
+    return res
+end
+
+-- Fast stack implementation.
```

Comment:
Proposed to move to a separate module for a readability reasons.

Reply:
If you do not insist, I do not want to. I do not have an intention to write a full-featured data structure. Otherwise, it does not deserve a separate module.